contracts/Autopay.sol

```diff
@@ -440,6 +469,18 @@ contract Autopay is UsingTellor {
             _rewardAmount = _feed.details.balance;
             _feed.details.balance = 0;
         }
+        // Adjust currently funded feeds
+        if (_feed.details.balance < _feed.details.reward) {
```

Comment:
we could save on gas by changing line 465 to `if (_feed.details.balance > _feed.details.reward) {`

and moving the code from lines 474-482 to the else statement on line 468